module/documents/actor/actor.mjs

```diff
@@ -3341,7 +3341,7 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
     const fill = CONFIG.DND5E.tokenHPColors[key];
 
     for ( const token of tokens ) {
-      if ( !token.object?.visible || !token.object?.renderable ) continue;
```

Comment:
We removed `renderable` in core `ActiveEffect#_displayScrollingStatus` with V12 Stable 4. Removing `renderable` makes the combat scrolling text appear when the token config is open, which wasn't the case previously.